app/views/main_content_panel.py

```diff
@@ -1604,6 +1619,43 @@ def _do_upload_list_rentry(self) -> None:
                 text="Failed to upload exported active mod list to Rentry.co",
             )
 
+    def _do_open_app_directory(self) -> None:
+        app_directory = os.getcwd()
+        logger.info(f"Opening app directory: {app_directory}")
+        platform_specific_open(os.getcwd())
+
+    def _do_open_settings_directory(self) -> None:
+        settings_directory = AppInfo().app_storage_folder
+        logger.info(f"Opening settings directory: {settings_directory}")
+        platform_specific_open(settings_directory)
+
+    def _do_open_rimsort_logs_directory(self) -> None:
+        logs_directory = AppInfo().user_log_folder
+        logger.info(f"Opening RimSort logs directory: {logs_directory}")
+        platform_specific_open(logs_directory)
```

Comment:
This is fine for now, but should move away from using AppInfo where possible. In its current state, it forces a weird non-pep approved import and init order of the modules due to weird dependency chains.

Reply:
Thanks for pointing that out, will look into it going forward